src/nodes/delta/approximations/extended.jl

```diff
@@ -0,0 +1,15 @@
+using Parameters
+
+export DeltaExtended, ET
+
+@with_kw struct DeltaExtended{T}
+    inverse::T = nothing
+end
```

Comment:
as an example of my comment on adding the `Parameters` package in `Project.toml` file. Can we perhaps also do something like this to limit the number of dependencies in `ReactiveMP`: 
```julia
struct DeltaExtended{T}
    inverse::T
end
DeltaExtended(; inverse=nothing) = DeltaExtended(inverse)
```

Reply:
I think both are fine. I like the `Parameters` package, but I don't see it has been used anywhere except Delta nodes. @bvdmitri should we better remove it?